public/mergify-configuration-schema.json

```diff
@@ -78,6 +78,12 @@
         "$ref": "#/definitions/QueueRule"
       }
     },
+    "priority_rules": {
+      "type": "array",
+      "items": {
+        "$ref": "#/definitions/PriorityRule"
+      }
```

Comment:
Missing description?

Reply:
No it's already defined, line 579, since it was used in the queue rules

Reply:
For example, I don't see anywhere the sentence "The rule with the highest priority value wins." in the new code path?

Reply:
I don't think that's a valid reason not to include the description. This file is going to be used by the dashboard for example to generate content.

Reply:
I have no idea, i just copied what was already done. I'll add a description.

Reply:
Done.